src/app/clusters/pump-configuration-and-control-server/pump-configuration-and-control-server.cpp

```diff
@@ -287,14 +323,28 @@ void emberAfPumpConfigurationAndControlClusterServerInitCallback(EndpointId endp
     emberAfDebugPrintln(
         "Constant Pressure %s",
         FeatureSupportedDebugString(endpoint, Attributes::MinConstPressure::Get, Attributes::MaxConstPressure::Get));
-    emberAfDebugPrintln("Constant Proportional Pressure %s",
+    emberAfDebugPrintln("PCC Server: Constant Proportional Pressure %s",
                         FeatureSupportedDebugString(endpoint, Attributes::MinCompPressure::Get, Attributes::MaxCompPressure::Get));
-    emberAfDebugPrintln("Constant Flow %s",
+    emberAfDebugPrintln("PCC Server: Constant Flow %s",
                         FeatureSupportedDebugString(endpoint, Attributes::MinConstFlow::Get, Attributes::MaxConstFlow::Get));
-    emberAfDebugPrintln("Constant Temperature %s",
+    emberAfDebugPrintln("PCC Server: Constant Temperature %s",
                         FeatureSupportedDebugString(endpoint, Attributes::MinConstTemp::Get, Attributes::MaxConstTemp::Get));
-    emberAfDebugPrintln("Constant Speed %s",
+    emberAfDebugPrintln("PCC Server: Constant Speed %s",
                         FeatureSupportedDebugString(endpoint, Attributes::MinConstSpeed::Get, Attributes::MaxConstSpeed::Get));
+
+    status = Attributes::ControlMode::Get(endpoint, &controlMode);
+    if (status == EMBER_ZCL_STATUS_UNSUPPORTED_ATTRIBUTE)
+    {
+        emberAfDebugPrintln("PCC Server: ControlMode attribute not available");
+        isControlModeAvailable = false;
```

Comment:
What if we have two instances of this cluster on different endpoints?

Reply:
That is most certainly a case I did not think of. I need to have a set of "isAttributeAvaialble" for each endpoint

Reply:
I think a "runtime" check is simplest, fwiw...

Reply:
I have moved the checks into the updateAttributeLinks rather than trying to determine it in the init function, so it should be able to deal with multiple instances on different endpoints now.